pkgs/development/python-modules/paho-mqtt/default.nix

```diff
@@ -0,0 +1,34 @@
+{ buildPythonPackage
+, lib
+, isPyPy
+, isPy27
+, fetchFromGitHub
+, pylama
+, pytest
+, pytestrunner
+, mock
+}:
+buildPythonPackage rec {
+  name = "paho-mqtt-${version}";
+  version = "1.4.0";
+
+  disabled = isPyPy;
+
+  # fetch from GitHub so we can run the tests
+  src = fetchFromGitHub {
+    owner = "eclipse";
+    repo = "paho.mqtt.python";
+    rev = "v${version}";
+    sha256 = "1xg9ppz2lqacd9prsrx93q2wfkjjyla03xlfw74aj1alz9ki5hrs";
+  };
+
+  buildInputs = [ pylama ];
```

Comment:
I suppose this is used for tests, or not? If so, it should be in checkInputs. Also, I think we can avoid including it altogether as it doesn't really make much sense having it running in our build.

Reply:
pylama is a linter - right, we don't necessarily need to lint the codebase, although I'd still suggest to keep running the unit tests ;-)